lms/static/js/Markdown.Converter.js

```diff
@@ -50,7 +51,7 @@ else
 // file before uncommenting it.
 //
 
-(function() {
+(function() {

```

Comment:
no change but still shows up as change. Take a look

Reply:
I'm unable revert this fix for some reason. Has to do something with the line endings.